doc/docs/master/how-tos/use-jupyterhub/using-jupyterhub.md

```diff
@@ -0,0 +1,236 @@
+# Using `python-pachyderm` in JupyterHub
+
+If you deployed JupyterHub as described in [Deploy Pachyderm with JupyterHub](../../deploy-manage/deploy/deploy-pachyderm-jupyterhub.md),
+the `python-pachyderm` client is preinstalled in your JupyterHub instance
+This section describes a few basic operations that you can execute from JupyterHub
+to interact with Pachyderm.
+
+After you log in, use the [python-pachyderm](https://pachyderm.github.io/python-pachyderm/python_pachyderm.m.html#header-functions)
+client API to manage Pachyderm directly from your Jupyter notebook.
+
+You need to create a new Notebook and add your code in a new cell.
+To run your code, click the <i class="fa fa-play" aria-hidden="true"></i> icon.
```

Comment:
Does this icon have a name in JH. like the "run" button or something? If so should be:

...click the run button (icon)... or something similar

Reply:
I'm just going to say "Run" because it looks like in different versions/flavors of JH, the button might look different.